src/training/method.ts

```diff
@@ -1,8 +1,20 @@
+import { Dayjs } from "dayjs";
+
+const dayjs = require('dayjs');
```

Comment:
```suggestion
import dayjs from "dayjs";
```
とかでdayjs使えたりしないですか？
(TSPlayGroundだとrequire()が必要かもしれません)
この辺を踏まえて、require()ってなんのために必要か、調べておいてください。

Reply:
require()を使わない場合、デフォルトのインポートがesModuleInteropではないためエラーになりました。
調べたことは以下の通りです。
> training配下でコンパイルするとエラーになる。
> エラーログから、esModuleInteropのフラグがtrueになっていないことが原因だと考えられる。
> tsconfigを確認するとフラグがfalseになっていることから、tsconfigの設定値を引き継げていないと考えられる。

Reply:
口頭で確認した通りです。(特に後続対応は不要)